Makefile

```diff
@@ -18,6 +18,7 @@ DOCKER_RUN_LIBCOMPOSE := docker run --rm -it --privileged $(LIBCOMPOSE_ENVS) $(L
 default: binary
 
 all: build
```

Comment:
Do we still need this build dependency here? I think it'll be run by `make.sh` right ?